src/problem2.rb

```diff
@@ -0,0 +1,22 @@
+
+# Given an integer array of size N,
+# identify the largest 2 numbers in this array. Do not sort the array.
+class Problem2
+  # @param [Array] numbers
+  # @return [Array]
+  def self.largest_two_of(numbers)
+    return nil if numbers.nil? || numbers.empty?
```

Comment:
This checks are not actually required, see [yagni](https://www.martinfowler.com/bliki/Yagni.html)